packages/node_modules/@webex/redux-module-conversation/src/actions.test.js

```diff
@@ -308,10 +308,8 @@ describe('redux-module-conversation actions', () => {
       };
       const parentId = '77981d00-3061-11ea-bc29-4b6836e60516';
 
-      return store.dispatch(actions.handleAdaptiveCardSubmitAction(url, actionInput, parentId, mockSpark))
-        .then(() => {
-          expect(store.getActions()).toMatchSnapshot();
-        });
+      store.dispatch(actions.handleAdaptiveCardSubmitAction(url, actionInput, parentId, mockSpark));
+      expect(store.getActions()).toMatchSnapshot();
```

Comment:
Since this action has multiple promises, it needs to be in the format it previously was for the test to actually test.